omnibus/config/projects/agent.rb

```diff
@@ -118,16 +118,26 @@
 # Datadog agent
 dependency 'datadog-agent'
 
+python_runtimes = ENV['PYTHON_RUNTIMES'].split(',')
+
 # Additional software
 if windows?
-  dependency 'cacerts_py2'
-  dependency 'cacerts_py3'
+  dependency 'cacerts_py2' if python_runtimes.include? "2"
```

Comment:
if we go down the path of the project variable, this use case would be `if project.python_runtimes.include? "2"` (it applies to any software def below)